### PR TITLE
Add Note on Previews to PR Template

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Adds a link to [my preview image generator](https://petershaggynoble.github.io/MDI-Sandbox/simpleicons/preview/) to our pull request template.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 <!--
-Before opening your pull request, have a quick look at our contribution guidelines: https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+Before opening your pull request, have a quick look at our contribution guidelines:
+https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+-->
+
+<!--
+Add a preview image of your submission using
```
Comment (ericcornelissen) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would:

- Just include it in the same comment.
- Add a colon at the end, like the existing line.
- Either have an extra empty line in both or in neither.

```suggestion

Add a preview image of your submission using:

```

Comment (ericcornelissen) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would reword this using "consider" so  contributors don't feel obliged to use it. For now I would like to keep this as an optional thing so we can get some feedback on the system. Perhaps even ask for feedback here? What do you think @PeterShaggyNoble?

```suggestion
-->

<!--
Consider adding a preview image of your submission using
```

**Comment by PeterShaggyNoble** [comment]
Thanks for the detailed feedback, Eric. Agree with all those changes, particularly the wording - the less work there is for contributors putting PRs together, the better - definitely don't want people feeling obliged to add them.

> Also, what would you say about adding a note about this in the Contributing Guidelines?

Good idea. Although, I'd hold off until we've had a bit of feedback and I've worked the last few kinks out. Or maybe even until we get the generator integrated into the new site or set up in its own SI repo, as we discussed.
